server/apps/postgres-app/src/main/java/org/apache/james/modules/data/PostgresDLPConfigurationStoreModule.java

```diff
@@ -31,6 +31,7 @@
 import com.google.inject.TypeLiteral;
 import com.google.inject.multibindings.Multibinder;
 
+//FIXME - I don't see what this has to do with postgres when it pulls stuff from data-cassandra ...
```

Comment:
I pulls from cassandra the event serialisation.

We could argue that `org.apache.james.dlp.eventsourcing.cassandra.DLPConfigurationModules` could be relocated in `data-library` where both the PG and the Cassandra implen can depend on it.

Reply:
>We could argue that org.apache.james.dlp.eventsourcing.cassandra.DLPConfigurationModules could be relocated in data-library where both the PG and the Cassandra implen can depend on it.

sounds good to me